### ISSUE TITLE
Add inputAttributes support for Select type

### ISSUE THREAD

**Issue by pmk65** [json-editor/json-editor#256]
Same as for string/input.

**Comment by pmk65** [comment]
@schmunk42 Should we prevent the "inputAttributes" functionality from setting keys used internally ("name" and "type") by JSON-Editor and those set using other options ("min", "max" is set using the options "maximum" and "minimum". And "pattern" is also set using other options. Not sure if there's more.) ?
As changing those might prevent the form from working correctly.

**Comment by schmunk42** [comment]
Hmm, basically this sounds like a good idea, but I don't know how many options we'd have to blacklist.
Would be rather easy to implement?

**Comment by pmk65** [comment]
For **_select_** there's just two: **type** and **name**
For **_string_** there's these: **name**, **type**, **maxlength**, **pattern**, **readonly**, **min**, **max** and **step**.

It just requires a single line of change in the code, from:

````javascript
     if (this.schema.options && this.schema.options.inputAttributes) {
      var inputAttributes = this.schema.options.inputAttributes;
      for (var key in inputAttributes) {
        if (inputAttributes.hasOwnProperty(key)) {
          this.input.setAttribute(key, inputAttributes[key]);
        }
      }
    }
````
to:

````javascript
     if (this.schema.options && this.schema.options.inputAttributes) {
      var inputAttributes = this.schema.options.inputAttributes;
      for (var key in inputAttributes) {
        if (inputAttributes.hasOwnProperty(key) && ['name','type'].indexOf(key) == -1) {
          this.input.setAttribute(key, inputAttributes[key]);
        }
      }
    }
````
And we can then fill in more if we discover some is missing.

**Comment by pmk65** [comment]
Fixed in #257
